### PR TITLE
Updates for LSST renaming

### PR DESCRIPTION
**Pull request by sethdigel**
The changes are in Section 4.  The version number is proposed to be bumped from 1.0.0 to 1.1.0.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
 a page break (often the error message does not identify this as the underlying problem). In such cases, then simplest solution is to find the
 offending link and enclose it within \verb|\mbox{}|. Note that the {\tt hypperref} package has a `draft' option that will disable all linking; this can be useful to get a draft paper to compile when La\TeX\ has split such a link. This is a better solution while a paper is in development, since the offending link may no longer fall on a page break once all material has been added to the paper.
 
-\section{How to Refer to DESC and LSST} 
+\section{How to Refer to DESC, LSST, the Rubin Observatory, the Telescope, and Camera}
```
Comment (rmandelb) [inline main.tex:143]:
```suggestion
\section{How to Refer to DESC, LSST, Rubin Observatory, the Telescope, and Camera} 
```
(the renaming guide says "Neither Vera C. Rubin Observatory nor Rubin Observatory is preceded by “the,” unless there is an object after the name")

File: main.tex [hunk 2]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
 a page break (often the error message does not identify this as the underlying problem). In such cases, then simplest solution is to find the
 offending link and enclose it within \verb|\mbox{}|. Note that the {\tt hypperref} package has a `draft' option that will disable all linking; this can be useful to get a draft paper to compile when La\TeX\ has split such a link. This is a better solution while a paper is in development, since the offending link may no longer fall on a page break once all material has been added to the paper.
 
-\section{How to Refer to DESC and LSST} 
+\section{How to Refer to DESC, LSST, the Rubin Observatory, the Telescope, and Camera} 
+
+This section is informed by the name usage guide for the Vera C. Rubin Observatory\footnote{\url{https://project.lsst.org/documents/name-use-guide}} and guidelines developed among the LSST science collaborations.
```
Comment (rmandelb) [inline main.tex:145]:
```suggestion
This section is informed by the name usage guide for Vera C. Rubin Observatory\footnote{\url{https://project.lsst.org/documents/name-use-guide}} and guidelines developed among the LSST science collaborations\footnote{\url{http://bit.ly/RubinLSSTSCsnaming}}.
```

File: main.tex [hunk 3]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
...
+
+\item{If LSST has been defined previously the first reference to DESC would be to the ``Rubin Observatory LSST Dark Energy Science Collaboration (DESC)''.}
+
+\item{Subsequently use ``LSST DESC'' or ``DESC'' for the collaboration.}
+
+\item{At the first reference to the collaboration in the main text include a footnote with the URL for the DESC Web site, \verb|\footnote{\url{http://lsstdesc.org}}|.}
+
+\item{Depending on the context, you may also want to cite the \href{http://adsabs.harvard.edu/abs/2012arXiv1211.0310L}{DESC white paper}, or have a footnote to the DESC Science Roadmap via \\\verb|\footnote{\url{https://doi.org/10.5281/zenodo.3547567}}|}. % \\although both documents are somewhat dated.}
+
+\item{The first reference to the observatory by itself should be to the ``Vera C. Rubin Observatory".  Subsequent references should be either ``Rubin Observatory" or ``Rubin" (not VRO).}
```
Comment (rmandelb) [inline main.tex:160]:
```suggestion
\item{The first reference to the observatory by itself should be to ``Vera C. Rubin Observatory''.  Subsequent references should be either ``Rubin Observatory'' or ``Rubin'' (not VRO).}
```

File: main.tex [hunk 4]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
...
+
+\item{The telescope should be referred to as the ``Simonyi Survey Telescope at Rubin Observatory''.}
 
-\item{LSST should be spelled out at the first use (in both the abstract and the main text of the paper) and a footnote provided to the LSST Web site, \verb|\footnote{\url{http://www.lsst.org}}|}
+\item{The camera should be referred to ``Rubin Observatory LSST Camera'' at the first reference.  Subsequently, use ``LSST Camera''.}
```
Comment (rmandelb) [inline main.tex:164]:
```suggestion
\item{The camera should be referred to as ``Rubin Observatory LSST Camera'' at the first reference.  Subsequently, use ``LSST Camera''.}
```

**Review by rmandelb — CHANGES_REQUESTED** [review]
I made some suggestions, most of which are for consistency with the name usage guide and one in the spirit of providing a link to more info (adding the link to the full guidelines for the SCs).

They are all in the form of suggestions on the PR, so if you like them you can click a button to commit the suggestion directly on the web interface.

Thank you for putting this together!

File: main.tex (hunk 2 above)
Comment (sethdigel) [reply main.tex:145]: Thank you, Rachel.  Do we expect that the guidelines developed among the LSST science collaborations will be staying in that Google doc or promoted to something with a more permanent looking URL?

**Comment by sethdigel** [comment]
Thanks a lot for the careful reading and suggestions, Rachel!

File: main.tex (hunk 2 above)
Comment (rmandelb) [reply main.tex:145]: That's meant to be the permanent home.

**Review by rmandelb — DISMISSED** [review]

**Comment by sethdigel** [comment]
Thanks again, Rachel.  I somehow 'dismissed' your 'stale review'.

File: main.tex [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 \usepackage{multirow}
 \usepackage{datetime2}
-\newcommand*{\version}{Version 1.0.0 (2018-04-20)} %{Version \today}
+\newcommand*{\version}{Version 1.1.0 (2020-04-16)} %{Version \today}
```
Comment (sethdigel) [inline main.tex:5]:
```suggestion
\newcommand*{\version}{Version 1.1.0 (2020-04-21)} %{Version \today}
```

File: main.tex [hunk 6]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
...
 
 \begin{itemize}
-\item{DESC should be spelled out at the first use (in both the abstract and the main text of the paper) and a footnote added with the URL for the DESC Web site, \verb|\footnote{\url{http://lsst-desc.org}}|.}
-  
-\item{Depending on the context, you may also want to cite the \href{http://adsabs.harvard.edu/abs/2012arXiv1211.0310L}{DESC white paper}, or have a footnote to the DESC Science Roadmap via \\\verb|\footnote{\url{http://lsst-desc.org/sites/default/files/DESC_SRM_V1.pdf}}|, \\although both documents are somewhat dated.}
+\item{At the first use (in both the abstract and the main text of the paper) DESC should be spelled out with the observatory and survey:  ``Rubin Observatory Legacy Survey of Space and Time (LSST) Dark Energy Science Collaboration (DESC)''.}
+
+\item{Where LSST is first referenced in the main text include a footnote to the LSST Web site,\\ \verb|\footnote{\url{http://www.lsst.org}}|.}
+
+\item{If LSST has been defined previously the first reference to DESC would be to the ``Rubin Observatory LSST Dark Energy Science Collaboration (DESC)''.}
```
Comment (sethdigel) [inline main.tex:152]:
```suggestion
\item{If LSST has been defined previously the first reference to DESC would be to ``Rubin Observatory LSST Dark Energy Science Collaboration (DESC)''.}
```

File: main.tex [hunk 7]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
...
+
+\item{Subsequently use ``LSST DESC'' or ``DESC'' for the collaboration.}
+
+\item{At the first reference to the collaboration in the main text include a footnote with the URL for the DESC Web site, \verb|\footnote{\url{http://lsstdesc.org}}|.}
+
+\item{Depending on the context, you may also want to cite the \href{http://adsabs.harvard.edu/abs/2012arXiv1211.0310L}{DESC white paper}, or have a footnote to the DESC Science Roadmap via \\\verb|\footnote{\url{https://doi.org/10.5281/zenodo.3547567}}|}. % \\although both documents are somewhat dated.}
+
+\item{The first reference to the observatory by itself should be to ``Vera C. Rubin Observatory''.  Subsequent references should be either ``Rubin Observatory'' or ``Rubin'' (not VRO).}
```
Comment (sethdigel) [inline main.tex:160]:
```suggestion
\item{The first reference to the observatory by itself should be to ``Vera C. Rubin Observatory''.  Subsequent references should be either ``Rubin Observatory'' or ``Rubin'' (not VRO, and not preceded with `the').  }
```

File: main.tex [hunk 8]
```diff
@@ -140,14 +140,28 @@ \subsection{Active hyperlinks}
...
+
+\item{The telescope should be referred to as the ``Simonyi Survey Telescope at Rubin Observatory''.}
 
-\item{LSST should be spelled out at the first use (in both the abstract and the main text of the paper) and a footnote provided to the LSST Web site, \verb|\footnote{\url{http://www.lsst.org}}|}
+\item{The camera should be referred to as ``Rubin Observatory LSST Camera'' at the first reference.  Subsequently, use ``LSST Camera''.}
```
Comment (sethdigel) [inline main.tex:164]:
```suggestion
\item{The camera should be referred to as ``Rubin Observatory LSST Camera'' at the first reference.  Subsequently, use the ``LSST Camera''.}
```

**Pull request merged**